src/index.html

```diff
@@ -1,32 +1,28 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Document</title>
-    <link
-      rel="stylesheet"
-      href="style.css"
-    />
-  </head>
-  <body>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Search Bar for Airbnb</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+  <form class="search-bar search-bar--big" data-ga="big">
```

Comment:
The `data-ga` attribute should be renamed to `data-qa` to meet the task requirements and ensure consistency, especially if these attributes are used for testing or tracking purposes.